src/Interfaces/AbstractZonotope.jl

```diff
@@ -407,15 +407,32 @@ factors are ``ξ_i ∈ \\{-1, 1\\}``.
 There are at most ``2^p`` distinct vertices. Use the flag `apply_convex_hull` to
 control whether a convex hull algorithm is applied to the vertices computed by
 this method; otherwise, redundant vertices may be present.
+
+We use a trick to speed up enumerating vertices of 2-dimensional zonotopic
+sets with all generators in the first quadrant or third quadrant (same sign).
+Namely, sort the generators in angle and add them clockwise in increasing
+order and anticlockwise in decreasing order.
 """
 function vertices_list(Z::AbstractZonotope{N};
                        apply_convex_hull::Bool=true) where {N<:Real}
     c = center(Z)
     G = remove_zero_columns(genmat(Z))
-    p = size(G, 2)
+    n, p = size(G)
     if p == 0
         return [c]
     end
+    if n == 2 && p > 2 && apply_convex_hull
+        if sum(abs, G) == abs(sum(G))
+            sorted_G = sortslices(G, dims=2,by=x->atan(x[2], x[1]))
+            index = ones(p, 2*p)
+            for i in 1:p
+                index[i, i+1:i+p-1] *= -1
```

Comment:
```suggestion
                index[i, i+1:i+p-1] .= -one(N)
```
This is cheaper than reading the 1 and multiplying with -1 (you know the result already).
Benchmark with `p = 20`:
```julia
julia> @time for i in 1:p
       index[i, i+1:i+p-1] .= -1.0
       end;
  0.000017 seconds (81 allocations: 3.469 KiB)

julia> @time for i in 1:p
       index[i, i+1:i+p-1] *= -1
       end;
  0.000028 seconds (81 allocations: 10.656 KiB)
```